__tests__/test-httpClient.js

```
@@ -0,0 +1,101 @@
+/* eslint-env node, browser, jasmine */
+import { jest } from '@jest/globals'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)